<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistControls</class>
 <widget class="QWidget" name="PlaylistControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>327</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>327</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="playlist_controls_layout">
     <item>
      <layout class="QHBoxLayout" name="playlist_controls_button_layout">
       <item>
        <widget class="QPushButton" name="playlist_controls_new">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Cr√©er un nouvelle playlist ou vider la playlist actuelle.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/icons.qrc">
           <normaloff>:/playlist/new.png</normaloff>:/playlist/new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playlist_controls_load">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Ouvrir une playlist ...</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/icons.qrc">
           <normaloff>:/playlist/open.png</normaloff>:/playlist/open.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playlist_controls_save">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Sauvegarder la playlist actuelle ...</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/icons.qrc">
           <normaloff>:/playlist/save.png</normaloff>:/playlist/save.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="playlist_controls_search">
         <property name="toolTip">
          <string>Recherche dans la playlist</string>
         </property>
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="playlist_controls_view">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
